src/components/Node/Node.module.scss

```diff
@@ -10,7 +10,7 @@
   justify-content: center;
 
   .icon {
-    font-size: 1.2em;
+    font-size: 1.7em;
     color: $black;
     padding-right: 5px;
   }
```

Comment:
As above, we need to fix a few styles to get the node header more in line with the design. First, replace this with:

```css
.icon {
  color: $black;
  margin-right: 8px;
}
```

Then add:

```css
.statusIcon {
  margin-left: 20px;
}
```

Then replace the `StatusIcon` function in the Node component with the following:

```js
const StatusIcon: FC<StatusIconProps> = ({ status }) => {
  if (status === null) return null;
  const icon = status ? faCheckCircle : faTimesCircle;
  return <FontAwesomeIcon icon={icon} className={styles.statusIcon} />;
};
```

These changes affect the comments section in the ExpandedNode component, so we also need make a few small changes to fix them. First, add a `<div>` around the label and button for the comments like so:

```js
<div>
  <label>Comments:</label>
  <button
    className={`${indexStyles.button} ${styles.defaultTextButton}`}
    onClick={(e): void => {
      e.preventDefault();
        if (!comments.includes(defaultText)) setComments(comments + defaultText);
    }}
  >
    Use Default Text
  </button>
</div>
```

And finally, fix the `.commentsForm` css selector in the stylesheet for the ExpandedNode component as follows:

```css
.commentsForm {
  margin-top: 15px;

  > div > label {
    font-weight: bold;
  }
}
```